hack/functests.sh

```diff
@@ -21,7 +21,7 @@ set -e
 
 DOCKER_TAG=${DOCKER_TAG:-devel}
 DOCKER_TAG_ALT=${DOCKER_TAG_ALT:-devel_alt}
-KUBEVIRT_E2E_PARALLEL_NODES=${KUBEVIRT_E2E_PARALLEL_NODES:-6}
+KUBEVIRT_E2E_PARALLEL_NODES=${KUBEVIRT_E2E_PARALLEL_NODES:-3}
```

Comment:
I believe the images were updated to be more dynamic so that should be ok - yes I can include a reduction in memory in the final PR - I am just doing some testing currently. 

It looks like the network lane will go for 2 hours to 2 hours  45 mins runtime with this change...

Reply:
Well, we better have stabilized lane that take more time then have it flaky and retest it :sweat_smile: 
We will keep removing redundent tests and optimize the lane.